<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/Logo-resized.png</normaloff>:/icons/Logo-resized.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 5px; /* по умолчанию бороздка расширяется до размеров ползунка. задав высоту она принимает фиксированный размер */
     /*background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);*/
	background-color: #BAE1ED;
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
     /*background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);*/
	background-color: #12B0E8;
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -8px 0; /*рукоятка располагается по умолчанию в прямоугольнике содержимого бороздки. Расширяется наружу от бороздки */
     border-radius: 9px;
 }

 QSlider::handle:horizontal:hover{
	background-color: #128CE8;
	 border: 1px solid #454545;
}

/* После ползунка*/
QSlider::add-page:horizontal {
     border: 1px solid #999999;
     height: 5px; 
	 background-color: #81E1ED;
     margin: 2px 0;
 }

/* Перед ползунком*/
 QSlider::sub-page:horizontal {
     border: 1px solid #999999;
     height: 5px; 
	 background-color: #BAE1ED;
     margin: 2px 0;
 }

/**/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/speedometer.png</normaloff>:/icons/speedometer.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Производительность</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TextLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
           </property>
           <property name="text">
            <string>Внимание</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Использовать количество потоков</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QSlider" name="CurrentThreadCountSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="CurrentThreadCountLabel">
              <property name="text">
               <string>авто</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Максимальное количество потоков</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QSlider" name="MaxThreadCountSlider">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>99</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="MaxThreadCountLabel">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabel_2">
           <property name="text">
            <string>Использование всех ресурсов системы может негативно 
сказаться на производительности системы.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/logging.png</normaloff>:/icons/logging.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Протоколирование</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="LoggingCheckBox">
             <property name="text">
              <string>Включить протоколирование</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Количество итераций не больше</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MaxIterCountEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>200</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
